navit/CMakeLists.txt

```diff
@@ -103,7 +103,13 @@ endif(SHARED_LIBNAVIT)
 if(NOT MSVC)
 	SET(NAVIT_LIBS ${NAVIT_LIBS} m)
 endif(NOT MSVC)
-target_link_libraries(${NAVIT_LIBNAME}  ${MODULES_NAME} ${NAVIT_SUPPORT_LIBS} fib ${NAVIT_LIBS} )
+
+if(__APPLE__)
+target_link_libraries(${NAVIT_LIBNAME}  ${MODULES_NAME} ${NAVIT_SUPPORT_LIBS} fib ${NAVIT_LIBS} "-framework Foundation") 
```

Comment:
This seems to generate compilation errors on WinCE

Reply:
This change has nothing to do with the PR. It is necessary to build on macos.
I'm not so familiar with Git, to separate it from the other changes.